dubbo-common/src/main/java/org/apache/dubbo/common/config/Environment.java

```diff
@@ -131,7 +131,7 @@ public synchronized CompositeConfiguration getPrefixedConfiguration(AbstractConf
             prefixedConfiguration.addConfiguration(propertiesConfiguration);
         } else {
             // The sequence would be: SystemConfiguration -> AbstractConfig -> AppExternalConfiguration -> ExternalConfiguration -> PropertiesConfiguration
-            // Config center has the highest priority
```

Comment:
The highest is right, needn't to modify it.

Reply:
OK, according to your suggestion, I will modify this problem, should I directly continue to modify under this PR (which will result in multiple commit logs), or launch a new PR to modify (this one will only have one commit log)?

Reply:
commit and push directly.

Reply:
ok, I have finished the modification

Reply:
Other also need hanlde.